sql/src/main/java/io/crate/metadata/Scalar.java

```diff
@@ -79,6 +80,8 @@
     public Symbol normalizeSymbol(Function symbol, TransactionContext transactionContext) {
         try {
             return evaluateIfLiterals(this, symbol);
+        } catch (UnsupportedFeatureException e) {
```

Comment:
> you mean you changed if for debugging purposes?

No, not for debugging purposes but the existing code can produce hard-to-debug errors because it swallows arbitrary exceptions. This is definitely not good practice.